lib/cog/protected_bundle_error.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Cog.ProtectedBundleError do
+  defexception [:message, :reason]
+
+  def exception(reason),
+    do: %__MODULE__{message: "Operation not permitted on a protected bundle: #{inspect reason}",
+                    reason: reason}
```

Comment:
Does this mean the the message here will end up being `Operation not permitted on a protected bundle: {:protected_bundle, "operable"}`. Seems like we'd really only need to send the bundle name arg as the `:protected_bundle` atom is sorta redundant.